src/builder.rs

```diff
@@ -82,9 +76,10 @@ impl Extra {
 /// Builder of [`DoubleArrayAhoCorasick`].
 pub struct DoubleArrayAhoCorasickBuilder {
     states: Vec<State>,
-    extras: [Extra; FREE_STATES as usize],
+    extras: Vec<Extra>,
     head_idx: u32,
     match_kind: MatchKind,
+    n_free_blocks: u32,
```

Comment:
```suggestion
    num_free_blocks: u32,
```

In this crate, valuables for numbers are indicated by `num_` such as `num_states`.
For consistency, `n_free_blocks` should be modified into `num_free_blocks`.